### ISSUE TITLE
Can't figure out how to make a Xamarin.Mac app

### ISSUE THREAD

**Issue by tpitman** [picoe/Eto#534]
I have tried the quick start and the tutorial without success. I then found a google search result that took me to a link on how to diagnose problems.

There was a link there that took me here:

https://github.com/picoe/Eto/wiki/Running-%26-Debugging-a-Mac-application-in-Xamarin-Studio

The problem is I don't see anywhere on the New Project list that is a Empty Xamarin.Mac project.

Please give me updated instructions...I have the Paid version of Xamarin that is the latest Xamarin Studio 6. 

Basically I want to make a Xamarin.Mac app using Eto.Forms.

**Comment by 0xFireball** [comment]
Xamarin.Mac is free now

**Comment by tpitman** [comment]
Thank you for your reply. Do you have any information on my question?

**Comment by manio143** [comment]
Just to make sure - you're doing it on a Mac?
You simply need to create an empty project. A console application is an empty project.

**Comment by tpitman** [comment]
This is on a Mac.

I tried following those instructions to create en empty console app and add the code. When I do that and hit debug it open up a terminal window and then I get an exception saying the target can't be loaded.

Yes, I changed the setting from executable to executable with GUI like the instructions said. I followed them step by step.

**Comment by manio143** [comment]
Tbh I don't have a Mac, can't test it.
But you should make sure you have pulled both Eto.Forms and the appropriate library for Mac (either Gtk or Cocoa)

**Comment by tpitman** [comment]
I simply installed the Eto Nuget packages:

Eto.Forms
Eto.Platform.XamMac2

**Comment by manio143** [comment]
Checkout http://www.nuget.org/packages/Eto.Platform.Mac.Template/
Maybe this will help.

**Comment by manio143** [comment]
Checkout http://www.nuget.org/packages/Eto.Platform.Mac.Template/
Maybe this will help.

**Comment by tpitman** [comment]
Sadly that doesn't work on Mac in Xamarin Studio and the template package is not compatible with Xamarin Studio 6. :-(

The current answer to that is that it is a lot of work because they changed everything (I assume Microsoft) and it will take a while to update the packages.

So what I really need is someone to give me very clear instructions on how to use the latest Xamarin Studio to make a Mac app project from scratch that can eventually be released to the app store using Eto forms.

Thank you for taking the time to answer even though you don't have a Mac.

**Comment by tpitman** [comment]
Oh, one more thing. It needs to be XamMac2 and not MonoMac because I don't want to have to have the user install Mono on their mac.

**Comment by cwensley** [comment]
@tpitman, hm you are right there is no longer an Empty application - bummer!  

Anyway, you can make it an empty application by deleting all files in the new application project except for Program.cs and Info.plist.  Then, follow the instructions on that wiki page to update the Info.plist and Program.cs.

Hope this helps!

**Comment by 0xFireball** [comment]
I tried doing it manually on Mac but couldn't get it to work with XamMac2. I have gotten it working on Windows and Linux manually (no template, just NuGet)

**Comment by tpitman** [comment]
What you suggest of adding the Cocoa template and then deleting the files that are not needed and still get the exception:

Could not detect platform. Are you missing a platform assembly?

**Comment by cwensley** [comment]
Heh, we're having two conversations about the same thing (;  Anyway, I've updated the wiki page with new instructions, hopefully that makes it work for you.  I totally forgot that 2.2 did not include Xamarin.Mac platforms in its platform auto-detection routine.

**Comment by tpitman** [comment]
Sorry, at first these where 2 different conversations. I did see the other answer and that solved it for those who didn't see that thread...

new Application(Eto.Platforms.XamMac2).Run(new MainForm());

**Comment by cwensley** [comment]
Marking as closed as it appears the issue is resolved.  Thanks again!
